ts/store/reducers/wallet/wallets.ts

```diff
@@ -146,7 +146,10 @@ const reducer = (
     case getType(paymentUpdateWalletPsp.failure):
       return {
         ...state,
-        favoriteWalletId: pot.toError(state.favoriteWalletId, action.payload),
+        favoriteWalletId: pot.toError(
+          state.favoriteWalletId,
+          Error(action.payload)
```

Comment:
for consistency you may want to change `favoriteWalletId` to `pot.Pot<Wallet["idWallet"], string>`